src/Tickets/Commerce/Status/Trashed.php

```diff
@@ -28,6 +28,19 @@ class Trashed extends Status_Abstract {
 	 */
 	const SLUG = 'trash';
 
+	/**
+	 * Gets the slug of this status in WordPress.
+	 *
+	 * Since this is a core status, we do need to match.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_wp_slug() {
```

Comment:
This should be unnecessary? It's identical to the [one in the abstract](https://github.com/the-events-calendar/event-tickets/pull/3489/files#diff-93d790a4d112780d1f419e952b6d8f36299dd3c503c04f6fe172b5ff5bddda1eR57).

Reply:
you are confusing it with get_slug

Reply:
Nope, you're right, sorry.